src/modules/likeCounter.js

```diff
@@ -0,0 +1,8 @@
+function likeCounter(item, data) {
+  const likes = data.filter((v) => v.item_id === item);
+  const count = likes.map((c) => c.likes);
+  const total = count.reduce((a, b) => a + b, 0);
+  return total;
+}
```

Comment:
- [ ] Since you are using ES6, It would be better to use arrow functions instead of the word `function`.